### ISSUE TITLE
Credentials not saved

### ISSUE THREAD

**Issue by ooglyhLL** [regseb/castkodi#63]
Lately I noticed that I have to reenter username/password in order to connect to my Kodi system. This was not required in the past. 
Could there have been a change in Kodi (Matrix) perhaps?

**Comment by regseb** [comment]
Hi,

Kodi username/password are no longer required in Cast Kodi since version _4.0.0_ (released in June 2019). What version of Cast Kodi are you using? Where do you fill in username/password?

**Comment by ooglyhLL** [comment]
Add-ons Manager says: 5.14.1
Whenever I restart Firefox, it appears that clicking on the toolbar icon I am getting a dialog asking for my credentials. Using Coreelec 19/Kodi (Matrix). Authentication is enabled, so I am not surprised that it does ask, just that it continues to do so.

**Comment by regseb** [comment]
There is no authentication form in the toolbar popup; I don't understand where this form comes from. 😕

Could you:
- take a screenshot of the authentication form;
- list the values of the Cast Kodi configuration;
- list the values of the _Settings_ / _Services_ / _Control_ page in Kodi.

**Comment by ooglyhLL** [comment]
* The credentials pop up overlays the Cast Kodi drop-down. A proper screenshot would always make both dialogs disappear.![cast_kodi_dialog](https://user-images.githubusercontent.com/10389424/128933480-107d8007-3c49-40b0-bb7a-3ac28c5a19b7.jpg)
* Settings:
   - allow automatic updates: default
   - run in private windows: on
   - configure one server: coreelec
   - add casted links in browser history: off
   - context menu: bookmark: off, others: on
   - YouTube: cast all videos of playlist
   - Select order of playlist: ask
   - optional permissions: both off
 * Kodi settings:
   - HTTP enabled
   - Port: 8080
   - Authentication required: on
   - SSL activated: no
   - Web interface: CoreELEC web interface - Chorus 2
 
Additional info: Reinstalling the add-on didn't change anything.

**Comment by regseb** [comment]
In the toolbar popup, there is a request to the Kodi web interface to know if it's accessible. This interface requires a connection and therefore Firefox displays the authentication form. I don't know why now this form appears. I reproduced the bug with Kodi 18 and 19. Maybe a change in Firefox.

While waiting to find a permanent solution, I have disabled the link to the Kodi web interface in Cast Kodi version _5.14.2_.
